packages/eslint-plugin-next/lib/rules/no-img-element.js

```diff
@@ -22,9 +22,15 @@ module.exports = {
           return
         }
 
+        if (node.parent?.parent?.openingElement?.name?.name === 'picture') {
+          return
+        }
+
         context.report({
           node,
-          message: `Do not use \`<img>\` element. Use \`<Image />\` from \`next/image\` instead. See: ${url}`,
+          message:
+            'Do not use `<img>` element. Use either Image from `next/image` or `<picture>` in conjunction with `<img>`. ' +
+            `See: ${url}`,
```

Comment:
```suggestion
          message: `Do not use \`<img>\` element. Use \`<Image />\` from \`next/image\` instead. See: ${url}`,
```
Let's leave this message as is since we don't want to recommend manual `<picture>` usage currently and instead want to investigate good defaults for `next/image` so it can handle the use case instead.